en/docs/administer/using-symmetric-encryption.md

```diff
@@ -1,15 +1,59 @@
-# Using Symmetric Encryption
+# Configurations Related to Symmetric Key Encryption
 
-With symmetric encryption a single key will be shared for encryption and decryption of information. Even though, WSO2 Identity Server uses [asymmetric encryption](../../administer/using-asymmetric-encryption) by default, you can switch to symmetric encryption. 
+This section explains the default configurations related to [symmetric key encryption](../../administer/symmetric-overview).
```

Comment:
IMO, it might be useful to mention that symmetric encryption is now the default on this page as well in case the user does not go to the overview page and lands here directly.